srcpkgs/bees/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'bees'
+pkgname=bees
+version=081a6af278fc2b4f33198e2ec0028f8383bb6d4a
```

Comment:
Use the versioned tags and not commits.
https://github.com/Zygo/bees/releases/tag/v0.6.5

Reply:
That tag fails to compile and all prior tags lack critical bug fixes. The commit I have chose to use is that of current master, which is stable as far as I can tell (Was tested on 3TB of data) and has no bugs to mention.

Reply:
then make them make a working release, void does not package git versions

Reply:
Alright, I'll attempt to contact them tomorrow and see if I can get them to post it as a release. Thank you for your help.